### PR TITLE
@bhoggard - Omit page and size from options passed

### PR DESCRIPTION
**Pull request by broskoski**
Here's a complicated explanation, get ready...... 

1. https://github.com/artsy/metaphysics/blob/master/schema/sale/index.js#L136 has default arguments and they were being passed down to the `gravity.all` loader (which is a little unnecessary, but ok).
2. We recently changed the way `total` gets called (which passes options down here https://github.com/artsy/metaphysics/blob/master/lib/all.js#L10 , in this case `page` and `size` where in the options (because they were default values). 
3. Because our cache uses args + path as the cache key, `total` was overwriting the first response's cache (which was identical to the `all` loader requesting the first page).

### CODE REVIEW

**Comment by broskoski** [comment]
TLDR; `total` doesn't need the `page` and 'size` arguments passed in from the outside, and it may interfere with cache setting, so omit them both.
